source/Meadow.Core/Cloud/HealthReporter.cs

```diff
@@ -98,6 +97,10 @@ public async Task Send()
 
             await service!.SendEvent(ce);
         }
+        catch (Exception ex)
+        {
+            Resolver.Log.Error($"HealthReporter unable to send data: {ex.Message}");
+        }
```

Comment:
we should throw after this

Reply:
also, even without this `RethrowUnhandledExceptions` should log the message from the exception as well.